examples/config.yml

```diff
@@ -27,6 +27,10 @@ lighthouse:
   # serve_dns optionally starts a dns listener that responds to various queries and can even be
   # delegated to for resolution
   #serve_dns: false
```

Comment:
Now that we have a `lighthouse.dns` parent, moving this to `lighthouse.dns.enabled` would help keep things clean. We'd need to add a deprecation notice and allow folks to use the old config path for the time being as well.

Reply:
I intentionally didn't do this to preserve compatibility, i'd rather not deprecate config options that hit a stable release. Are you sure you want me to do that?

Reply:
I'm fine not having you write the support for it, but I think we should be moving to keep config as clean as we can. Happy to write it up after this merge.